### ISSUE TITLE
SegaCD Pack is Zip Files

### ISSUE THREAD

**Issue by eatnumber1** [SmokeMonsterPacks/EverDrive-Packs-Lists-Database#147]
Hey, how is verification / pack building intended to work with the new Sega CD collection? My Sega CD collection is compressed using 7zip, so won't verify against the .zip hashes that are currently in [SegaCD.txt](https://github.com/SmokeMonsterPacks/EverDrive-Packs-Lists-Database/blob/master/EverDrive%20Pack%20SMDBs/SegaCD.txt).

Is it possible to get the collection done as hashes of the decompressed files?

**Comment by maniac79** [comment]
Hi - I have picked up working on this yesterday.
However, I don't own a MegaSD, so can't test myself. I'll be in touch when I have something ready.

**Comment by maniac79** [comment]
Hey again, @eatnumber1. :)
Feel free to test if this suits your needs:
https://github.com/maniac79/EverDrive-Packs-Lists-Database/blob/master/EverDrive%20Pack%20SMDBs/MegaSD%20SMDB.txt

As stated before I don't own a MegaSD myself yet, so I can't test for myself.
Feedback is welcome.

**Comment by eatnumber1** [comment]
Thanks!

I haven't tested it yet, but from just looking at it, it looks like what I'd want. The split layout that you have of one `.bin` per track even allows for cross-referencing the hashes against redump.org!

For example, the file `D:/ROMs/Sega CD/MegaSD/1 US - A-M/3 Ninjas Kick Back (USA)/3 Ninjas Kick Back (USA) (Track 01).bin` has a SHA-1 hash of `fecabc877f98ffa76ff88c322d1d8c18f86f19c9`, which matches track 1 on the game's page [on redump](http://redump.org/disc/22251/).

Some disc images won't have the exact same file structure (for instance, you could have one `.bin` for the entire disc), but that's a tooling problem. Either folks will need to convert to a multi-bin image, or the pack building tooling in this repo could possibly be improved to handle those cases.

As things stand today, I think that this new layout you have is the best way to present a CD-based pack.

**Comment by eatnumber1** [comment]
Just successfully built a pack using my personal Sega CD collection. One detail though, the paths in the pack include the `D:/ROMs` path prefix, which results in packs which get generated being found under a directory named `D:/ROMs`. You might want to strip that out of the paths.

Otherwise, your branch looks good to me :). Thanks for doing it!

**Comment by maniac79** [comment]
Thanks for the feedback! I have just corrected the path prefix - I should have checked this before committing the change. :-/

Just for clarification, although I believe you have figured that out already:
I took the redump.org set as base and removed all dumps which seem to be duplicates.
I tried to keep major revisions though, favorizing the latest version in the "core directories" (e.g. "1 US - A-M") - the earlier revisions are kept in "4 Betas, Demos, Prototypes, Revisions, etc/Revisions/".

Then I added files from the Everdrive Packs, e.g. the Unworked Designs translations, MD+ games, etc.

Lastly, I added some files myself which I hope are meaningful and easily obtainable for others as well, mainly speaking about the different versions of Timecop.

I hope to own a MegaSD one day so I can test it for myself. :)
